py/build.gradle

```diff
@@ -111,8 +107,8 @@ task distPython(type: Exec, dependsOn: checkPythonEnv) {
         }
 
         copy {
-            from "${configurations.compile.join(',')}"
```

Comment:
@mmalohlava please any ideas on this ? It would be good to make the build parallel together with your PR https://github.com/h2oai/sparkling-water/pull/195, it would move PySparkling build  a bit forward again!